src/js/components/List/List.js

```diff
@@ -640,9 +650,15 @@ const List = React.forwardRef(
                 let displayPinned;
                 if (isPinned) {
                   // Pinned icon and settings
-                  const Pin = theme.list.icons.pin;
+                  // If pinned is passed in dataTableColumns use Lock icon.
+                  const Icon = inDataTableColumns ? Lock : theme.list.icons.pin;
                   const pinSize = theme.list.item.pinned.icon.size;
                   const pinPad = theme.list.item.pinned.icon.pad;
+                  const iconColor = inDataTableColumns ? pinTextColor : null;
+
+                  orderProps = {
```

Comment:
Is the `onOrder + !isPinned` logic on line 499 not working as expected? I wouldn't have expected to need to add this here based on how the logic is written above.

Reply:
not sure Im following the question above in the `onOrder + !isPinned` we have the logic for the dragging so that is where draggable is set to `true` so we need to set to `false` when pinned is true.

Reply:
Right, I suppose what I meant is if an item `isPinned`, why did it get the draggable true applied to it? I would expect it to fail the `onOrder && !isPinned` condition